src/amplitude-snippet.js

```diff
@@ -4,6 +4,7 @@
   as.type = 'text/javascript';
   as.async = true;
   as.src = 'https://d24n15hnbwhuhn.cloudfront.net/libs/amplitude-2.4.1-min.gz.js';
+  as.onload = function() { window.amplitude.runQueuedFunctions(); };
```

Comment:
Chrome and Firefox definitely support it. It's part of the HTML 4.01 standard, and seeing as how IE9 has HTML5, it should be okay: http://www.w3schools.com/tags/ev_onload.asp